code/espurna/homeassistant.ino

```diff
@@ -296,6 +313,21 @@ void _haInitCommands() {
 
 // -----------------------------------------------------------------------------
 
+void _haLoop() {
+    if (_ha_send_config.empty()) return;
+
+    uint32_t client_id = _ha_send_config.front();
+    _ha_send_config.pop();
+
+    _haDumpConfig([client_id](String& output) {
+        output.replace(" ", "&nbsp;");
+        output.replace("\n", "<br />");
```

Comment:
Maybe this also can be done on the client.
And, after #1387 wsSend change, can even directly use JsonObject to create the data structure

Reply:
...definitely must be in the client bc `Text(...)` can auto convert newlines to the proper html (like the logging does)